cli/utilities/utils.py

```diff
@@ -437,3 +438,18 @@ def is_node_online(node):
         return True
     log.error(f"{node.hostname} is DOWN, Ping failed")
     return False
+
+
+def get_lvm_list(container_id, node, format="json"):
```

Comment:
change name to `get_lvm_on_osd_container`

Reply:
Ack. changed